### PR TITLE
Feature attack ability

### PR DESCRIPTION
**Pull request by aimalesa**
Attack ability implemented

### CODE REVIEW

**Review by jiggly-jo — CHANGES_REQUESTED** [review]
Change sword attack damage to 1.

File: Assets/SwordAttack.cs [hunk 1]
```diff
@@ -0,0 +1,62 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class SwordAttack : MonoBehaviour
+{    
+    public BoxCollider2D swordCollider;
+    public float damage = 3;
```
Comment (jiggly-jo) [inline Assets/SwordAttack.cs:8]: Change attack damage to 1

**Review by jiggly-jo — CHANGES_REQUESTED** [review]
Change attack damage to 1

File: Assets/SwordAttack.cs (hunk 1 above)
Comment (aimalesa) [reply Assets/SwordAttack.cs]: Damage has been reduced to 1.

**Comment by jiggly-jo** [comment]
good work

**Review by jiggly-jo — APPROVED** [review]
nice

**Pull request merged**